com.avaloq.tools.ddk.check.ui/plugin.xml_gen

```diff
@@ -1,215 +1,220 @@
 <?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Does any of this need merging with plugin.xml?

Reply:
I think it is better not to do that, so I have just added a comment in the commit.